service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandler.java

```diff
@@ -62,16 +63,22 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
 
     @EventListener
     public void notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(StandardDirectionsOrderIssuedEvent event) {
-        if (featureToggleService.isAllocatedJudgeNotificationEnabled(SDO)) {
-            EventData eventData = new EventData(event);
-            CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+        EventData eventData = new EventData(event);
+        CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+
+        if (hasAllocatedJudgeEmail(caseData.getStandardDirectionOrder())) {
             AllocatedJudgeTemplateForSDO parameters = standardDirectionOrderIssuedEmailContentProvider
                 .buildNotificationParametersForAllocatedJudge(eventData.getCaseDetails());
 
-            String email = caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor().getJudgeEmailAddress();
+            String email =  caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor().getJudgeEmailAddress();
 
             notificationService.sendEmail(STANDARD_DIRECTION_ORDER_ISSUED_JUDGE_TEMPLATE, email, parameters,
                 eventData.getReference());
         }
     }
+
+    private boolean hasAllocatedJudgeEmail(Order standardDirectionOrder) {
+        return isNotEmpty(standardDirectionOrder.getJudgeAndLegalAdvisor())
```

Comment:
This could be the allocated or temporary judge here as they have the option in this event, do we want to call the method hasJudgeEmail instead or what you think? (c2 and cmo always pull from allocated judge event so name for them would be as is) let me know what you think